php/tests/compatibility_test.sh

```diff
@@ -119,16 +119,16 @@ cd protobuf/php
 composer install
 
 # Remove implementation detail tests.
-tests=( array_test.php encode_decode_test.php generated_class_test.php map_field_test.php well_known_test.php )
-sed -i.bak '/php_implementation_test.php/d' phpunit.xml
-sed -i.bak '/generated_phpdoc_test.php/d' phpunit.xml
-sed -i.bak 's/generated_phpdoc_test.php//g' tests/test.sh
-sed -i.bak 's/generated_service_test.php//g' tests/test.sh
+tests=( EncodeDecodeTest.php GeneratedClassTest.php MapFieldTest.php RepeatedFieldTest.php WellKnownTest.php )
```

Comment:
I think you shouldn't change this file, because the script works on our previous code which doesn't have your change.

Reply:
What do you mean by previous code? I though that when I renamed `encode_decode_test.php` to `EncodeDecodeTest.php` that it should be changed everywhere where used.